pyjl/transpiler.py

```diff
@@ -18,6 +18,15 @@
     SMALL_USINGS_MAP,
 )
 
+from py2many.analysis import get_id, is_void_function
+from py2many.declaration_extractor import DeclarationExtractor
+from py2many.clike import _AUTO_INVOKED, class_for_typename
+from py2many.tracer import is_list, defined_before, is_class_or_module, is_enum
+
+from pyjl import julia_ast
```

Comment:
When we install py2many, the transpilers are put under py2many.
To allow for that, change this to
```py
try:
    from py2many.pyjl import julia_ast
except ImportError:
    from pyjl import julia_ast
```